server/doc/swdesign/README.md

```diff
@@ -818,6 +818,24 @@ Needs:
 - impl
 - utest
 
+#### Server receives agent node resource availability
```

Comment:
This requirement does only document the call of the update. You need another one, that explains how the agent's resource availability is updated inside the ServerState. You can check how it was done for the other methods from ServerState and AgentMap.